### ISSUE TITLE
Contact button in navbar 

### ISSUE THREAD

**Issue by Prashant7451** [yashikabhargava/WebX#76]
## Description
Contact button in navbar  is irresponsive .

## Do you want to work on this issue?
Yes

## How are you planning to resolve on this issue?
Contact button will be linked to contact us.

## Are you raising this issue under hacktoberfest? (Remove this section if you are not a hacktoberfest participant)
Yes, I'm raising this issue under hacktoberfest.

**Comment by yashikabhargava** [comment]
@Prashant7451 The PR for this issue was opened by @sohail60  and I merged it. I just saw that this issue was opened by you and I apologize for not assigning it to you.

@sohail60 This is not a good practice please do not open a PR until the issue has been assigned to you.

**Comment by heyvats21** [comment]
@yashikabhargava i have created a new Issue check it out.

**Comment by sohail60** [comment]
@yashikabhargava @Prashant7451 So sorry. I didn't know that before.
